src/components/profile/profile-image.tsx

```
@@ -0,0 +1,173 @@
+                    }
+                    {
+                        form.getAllErrors().form.errors[0]?.profileImageUrl?.map((error, index) => (
+                            <div key={`${error.path}-${index}`} className='text-red-500 text-sm mt-2'>
+                                {error.message}
+                            </div>
+                        ))
+                    }
+                </div>
+                <form.Subscribe
+                    selector={(state) => [state.canSubmit, state.isSubmitting]}
+                    children={([canSubmit, isSubmitting]) => (
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)